### ISSUE TITLE
Portable zip flagged as malicious

### ISSUE THREAD

**Issue by robgazy** [UltraStar-Deluxe/USDX#927]
UltraStarDeluxe-portable-2024.10.0.zip flagged as malicious

7/68 security vendors flagged this file as malicious
712f5cd4d60004c7702107b4e2d35febc503c5c579cf9c5f33993f84793672fd
UltraStarDeluxe-portable-2024.10.0.zip

![image](https://github.com/user-attachments/assets/e50ddeaf-f43a-4ebc-b440-7ab6e3155fab)

**Comment by LeighBicknell** [comment]
The installer too
![image](https://github.com/user-attachments/assets/f4f3b098-74ec-447f-8f41-bb3d5f17a778)

**Comment by s09bQ5** [comment]
Look what I found!!
https://docs.virustotal.com/docs/false-positive-contacts

**Comment by robgazy** [comment]
Windows Security blocks the file and will not install or extract. 

Threat Blocked: Severe
Detected: Trojan:Script/Wacatac.B!ml
Status: Quarantined
Quarantined files are in a restricted area where they can't harm your device. They will be removed automatically.
Details: This program is dangerous and executes commands from an attacker.
Affected items: file: C:\Users\xyza\Downloads\UltraStarDeluxe-portable-2024.10.0.zip

**Comment by s09bQ5** [comment]
Please submit the file at https://www.microsoft.com/en-us/wdsi/filesubmission for malware analysis. I can't. I don't have a Microsoft account.

**Comment by basisbit** [comment]
Duplicate of #722 and #368 . It is super annoying, that big antimalware development companies doesn't test their stuff properly to avoid false detections of small software projects. This is a common problem that many small projects have to deal with when using installer tools like NSIS and similar, just because sometimes malware developers choose to also use the same installer tools.

**Comment by basisbit** [comment]
> Please submit the file at https://www.microsoft.com/en-us/wdsi/filesubmission for malware analysis. I can't. I don't have a Microsoft account.

Done, and already got a response from a manual analyst:
> At this time, the submitted files do not meet our criteria for malware or potentially unwanted applications. The detection has been removed. Please follow the steps below to clear cached detections and obtain the latest malware definitions.
> 1. Open command prompt as administrator and change directory to c:\Program Files\Windows Defender
> 2. Run “MpCmdRun.exe -removedefinitions -dynamicsignatures”
> 3. Run "MpCmdRun.exe -SignatureUpdate"
> Alternatively, the latest definition is available for download here: https://docs.microsoft.com/microsoft-365/security/defender-endpoint/manage-updates-baselines-microsoft-defender-antivirus

So, hopefully in the next few days when people's computers update, it shouldn't falsely detect usdx installer or portable zip any more.

**Comment by barbeque-squared** [comment]
Thanks basisbit! I'll probably leave this ticket open until I can work the info in this ticket into the readme/wiki, as it appears to be a recurring issue. Maybe even figure out how to use the github issue templates.

Slightly off-topic / note to self: do any of these scanning services have some kind of API so that we can work at least the portable into the CI? Should be allowed to fail / pass with warnings (at least to start with) but some kind of scanning is still better than no scanning at all.

**Comment by cskinner74** [comment]
Two weeks later and still being flagged, even after performing the above update steps

**Comment by s09bQ5** [comment]
@cskinner74 is it still flagged as malicious? None of the 65 malware scanners used by VirusTotal currently flag the portable zip.

**Comment by cskinner74** [comment]
@s09bQ5 Yeah, Windows Defender is still alerting on it when trying to download.

**Comment by barbeque-squared** [comment]
Closing this as there's nothing we can do. It's just Windows things.
